node/src/main/scala/org/bitcoins/node/SpvNode.scala

```diff
@@ -15,16 +15,24 @@ import org.bitcoins.node.networking.peer.{
 import org.bitcoins.rpc.util.AsyncUtil
 
 import scala.concurrent.Future
-
-case class SpvNode(peer: Peer, chainApi: ChainApi)(
+import org.bitcoins.core.bloom.BloomFilter
+import org.bitcoins.core.p2p.FilterLoadMessage
+import org.bitcoins.core.p2p.NetworkPayload
+
+case class SpvNode(
+    peer: Peer,
+    chainApi: ChainApi,
+    callbacks: Vector[SpvNodeCallbacks] = SpvNodeCallbacks.empty,
+    bloomFilter: Option[BloomFilter] = None
```

Comment:
You may want to sync with block filters, or maybe use this node to actually download everything from the network

But you're right that right now there's probably no use cases where this would be none. Can change it

Reply:
Yeah, especially with the name of the class being `SpvNode` which implies bloom filters IMO.